components/apim_appgw/local.tf

```diff
@@ -0,0 +1,56 @@
+locals {
+  env = var.env == "ithc" || var.env == "perftest" ? "nonprod" : var.env
+
+  vnet_rg     = var.env == "ithc" || var.env == "perftest" ? "hmcts-hub-nonprod-int" : "ss-${var.env}-network-rg"
+  vnet_name   = var.env == "ithc" || var.env == "perftest" ? "hmcts-hub-nonprodi" : "ss-${var.env}-vnet"
+  subnet_name = "hub-appgw"
+
+  key_vault_resource_group = "cft-platform-${var.subscription}-rg"
+  key_vault_subscription   = var.key_vault_subscription
+
+  gateways = yamldecode(data.local_file.configuration.content).gateways
+
+  apim = {
```

Comment:
additionally can you check this? I dont believe actual route table is used/assigned to anywhere

Reply:
Thats right - the route table is being created but its not associated with any subnet, same on the CFT

Reply:
want to fix? :D

Reply:
sure - done :D